src/v2/api/services/DisciplineTeacherService.ts

```diff
@@ -290,4 +295,17 @@ export class DisciplineTeacherService {
     }
     await this.disciplineTeacherRepository.deleteById(disciplineTeacher.id);
   }
+
+  async checkSelectiveness (userId: string, disciplineId: string) {
```

Comment:
This method works only for disciplines of current semester, but surveys can be taken on previous disciplines too.